.jscsrc

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Is it ok to put comments in this file? If so, it'd be nice to add something along the lines of

    // For information on what these options mean, please refer to:
    //   http://jscs.info/rules

If comments aren't supported, though, it's fine to ignore this.